.github/actions/set_github_user/action.yml

```diff
@@ -0,0 +1,9 @@
+name: Set GitHub User
+description: 'Sets github user for a particular workflow'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        git config user.name paypalsdk
```

Comment:
We took this user from the Unified SDK - is this correct or do we want to still use braintreeps?